testing/MLDB-1253_concat_test.py

```diff
@@ -0,0 +1,102 @@
+#
+# MLDB-1253_concat_test.py
+# Mich, 2016-01-13
+# Copyright (c) 2016 Datacratic Inc. All rights reserved.
+#
+
+import json
+import traceback
+
+if False:
+    mldb = None
+
+_test_cases = []
+
+
+def log(thing):
+    mldb.log(str(thing))
+
+
+def perform(*args, **kwargs):
+    res = mldb.perform(*args, **kwargs)
+    assert res['statusCode'] in [200, 201], str(res)
+    return res
+
+
+def query(query):
+    res = perform('GET', '/v1/query', [['q', query], ['format', 'table']])
+    return json.loads(res['response'])
+
+
+def record_row(ds_url, row_name, cols):
+    perform('POST', ds_url + '/rows', [], {
+        'rowName' : row_name,
+        'columns' : cols
+    })
+
+
+def test_case(*args, **kwargs):
+    _test_cases.append(args[0])
+
+
+def run_tests():
+    success = 0
+    errors = 0
+    for test_case in _test_cases:
+        log('Starting test: ' + test_case.__name__)
+        try:
+            test_case()
+            log(test_case.__name__ + ' SUCCESS')
+            success += 1
+        except Exception:
+            log(test_case.__name__ + ' FAILED')
+            errors += 1
+            log(traceback.format_exc())
+
+    log("{}/{} test(s) passed".format(success, success + errors))
+    if errors:
+        raise Exception("{} test(s) failed".format(errors))
```

Comment:
@nicolaskruchten From the top of the file to here, these are functions that I reuse in most of my codes. I am considering creating an utility library. As for the test_case + run_tests functions, I tried to import unittest but it doesn't work because sys.argv is undefined. I may be worth it to spend more time to make it work.